src/tests/ftest/control/dmg_storage_query.py

```diff
@@ -169,6 +173,7 @@ def test_dmg_storage_query_device_health(self):
         if err:
             self.fail("Health info not as expected: {}".format(err))
 
+    @skipForTicket("DAOS-4286")
```

Comment:
DAOS-4286 is **Resolved**.  You can't skip a test for a ticket that has been resolved.